cardano-cli/src/Cardano/CLI/Read.hs

```diff
@@ -1028,3 +1031,6 @@ readSafeHash =
   Opt.eitherReader $ \s ->
     readHexAsSafeHash (Text.pack s)
       & first (Text.unpack . renderReadSafeHashError)
+
+scriptHashReader :: Opt.ReadM ScriptHash
```

Comment:
What does the error look like if it's an invalid script hash?

Reply:
I'm not sure.  It's a pre-existing function moved from `Cardano.CLI.EraBased.Options.Governance.DRep`

Reply:
It seems to not fail at all.

Reply:
It might be broken, but I'm merging this on the basis that it's not a regression.  We're just moving this flag from the `drep delegation-certificate` command which will soon be deleted.  Will need to follow up on whether this flag actually works.